ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CLAIMANT_RESPONSE_SPEC",
+    "CaseFieldID": "applicant1RepaymentOptionForDefendantSpec",
```

Comment:
Please don't forget to remove/overwrite this with my version once my changes get merged

Reply:
definitely will do it